src/components/SceneCard/SceneCard.styled.ts

```diff
@@ -0,0 +1,354 @@
+import {
+  Box,
+  Card,
+  CardActionArea,
+  CardContent,
+  CardMedia,
+  Chip,
+  Theme,
+  styled,
+} from "@mui/material"
+import { hexToRgba } from "../../utils/colors"
+
+const CardContainer = styled(Card)<{
+  withShadow?: boolean
+  withBorder?: boolean
+}>(
+  ({
+    theme,
+    withShadow,
+    withBorder,
+  }: {
+    theme: Theme
+    withShadow?: boolean
+    withBorder?: boolean
+  }) => ({
+    borderRadius: theme.spacing(2),
+    boxSizing: "border-box",
+    maxWidth: theme.spacing(43.125),
+    backgroundColor: "transparent",
+    position: "relative",
+    overflow: "hidden",
+    transition:
+      "transform 0.1s ease-in-out, box-shadow 0.3s ease-in-out, height 0.2s ease-in-out, min-height 0.2s ease-in-out",
+
+    ...(withBorder && {
+      "&::before": {
+        content: '""',
+        position: "absolute",
+        inset: 0,
+        borderRadius: theme.spacing(2),
+        background: "linear-gradient(to bottom, #FF2D55, #FFBC5B)",
+        mask: `linear-gradient(#fff 0 0) content-box, linear-gradient(#fff 0 0)`,
+        WebkitMask: `linear-gradient(#fff 0 0) content-box, linear-gradient(#fff 0 0)`,
```

Comment:
Not sure about it but do we need to add webkit? sometimes styled components make all the work for you